<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSLR_Shooter_Window</class>
 <widget class="QMainWindow" name="DSLR_Shooter_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>935</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DSLR Shooter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QWidget" name="imageContainer" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>&amp;Camera</string>
    </property>
    <addaction name="actionScan"/>
    <addaction name="actionSet_Camera"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Main</string>
    </property>
    <addaction name="action_Quit"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuTelescope">
    <property name="title">
     <string>&amp;Telescope</string>
    </property>
    <addaction name="actionConnectTelescope"/>
    <addaction name="action_Devices_Control_Panel"/>
    <addaction name="actionRemote_Control"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Win&amp;dows</string>
    </property>
    <addaction name="actionCamera_Information"/>
    <addaction name="actionCamera_Setup"/>
    <addaction name="actionSequences_Editor"/>
    <addaction name="actionGuider"/>
    <addaction name="actionFocusing"/>
    <addaction name="actionLogs_Messages"/>
    <addaction name="actionHistogram"/>
    <addaction name="actionHide_All"/>
    <addaction name="actionShow_All"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionDebayer"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCamera"/>
   <addaction name="menuTelescope"/>
   <addaction name="menuView"/>
   <addaction name="menuWindows"/>
  </widget>
  <widget class="QDockWidget" name="camera_information_dock">
   <property name="windowTitle">
    <string>Camera I&amp;nformation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QLabel" name="camera_infos">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>32</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="camera_setup_dock">
   <property name="windowTitle">
    <string>Camera Set&amp;up</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_8"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="sequencesDock">
   <property name="windowTitle">
    <string>Se&amp;quences Editor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5"/>
  </widget>
  <widget class="QDockWidget" name="guider_dock">
   <property name="windowTitle">
    <string>Guide&amp;r</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="0" column="0">
      <widget class="QPushButton" name="connectLinGuider">
       <property name="text">
        <string>Connect Guider</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="dither">
       <property name="text">
        <string>Dither</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="guiderStatus">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="focus_dock">
   <property name="windowTitle">
    <string>Focusin&amp;g</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="enable_focus_analysis">
       <property name="text">
        <string>Enable focus analysis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="focus_analysis_widgets" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="5" column="1">
         <widget class="QLabel" name="focus_analysis_best_hdf">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="focus_analysis_value">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="focus_analysis_best_hdf_label">
          <property name="text">
           <string>Best HDF value</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="focus_analysis_type">
          <property name="text">
           <string>HFD (lower is better)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Focusing History</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="focusing_select_roi">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Select ROI</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="focusing_clear_roi">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Clear ROI</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QCustomPlot" name="focusing_graph" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="shoot_toolbar">
   <property name="windowTitle">
    <string>Shooting</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShoot"/>
   <addaction name="actionStart_Sequences"/>
   <addaction name="actionPause_Shooting"/>
   <addaction name="actionStop_Shooting"/>
  </widget>
  <widget class="QDockWidget" name="histogram">
   <property name="windowTitle">
    <string>Histogram</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QCustomPlot" name="histogram_plot" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionConnect_LinGuider">
   <property name="text">
    <string>Connect LinGuider</string>
   </property>
  </action>
  <action name="actionDisconnect_LinGuider">
   <property name="text">
    <string>Disconnect LinGuider</string>
   </property>
  </action>
  <action name="actionDither">
   <property name="text">
    <string>Dither</string>
   </property>
  </action>
  <action name="actionScan">
   <property name="text">
    <string>&amp;Scan</string>
   </property>
  </action>
  <action name="actionSet_Camera">
   <property name="text">
    <string>Set &amp;Camera</string>
   </property>
  </action>
  <action name="actionSet_Preview_Interval">
   <property name="text">
    <string>Set &amp;Preview Interval...</string>
   </property>
  </action>
  <action name="actionClean_Log_Messages">
   <property name="text">
    <string>&amp;Clean Log Messages</string>
   </property>
  </action>
  <action name="actionOpen_Last_Images_Folde">
   <property name="text">
    <string>&amp;Open Last Images Folder</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionConnectTelescope">
   <property name="text">
    <string>&amp;Connect Server...</string>
   </property>
  </action>
  <action name="action_Devices_Control_Panel">
   <property name="text">
    <string>&amp;Devices</string>
   </property>
  </action>
  <action name="actionRemote_Control">
   <property name="text">
    <string>&amp;Telescope Control</string>
   </property>
  </action>
  <action name="action_LogMessages">
   <property name="text">
    <string>&amp;Log Messages</string>
   </property>
  </action>
  <action name="actionCamera_Information">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Camera Information</string>
   </property>
  </action>
  <action name="actionCamera_Setup">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera &amp;Setup</string>
   </property>
  </action>
  <action name="actionGuider">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Guider</string>
   </property>
  </action>
  <action name="actionFocusing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Focusing</string>
   </property>
  </action>
  <action name="actionLogs_Messages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Logs Messages</string>
   </property>
  </action>
  <action name="actionHide_All">
   <property name="text">
    <string>&amp;Hide All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionShow_All">
   <property name="text">
    <string>Sh&amp;ow All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+H</string>
   </property>
  </action>
  <action name="actionShoot">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="camera-photo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Shoot</string>
   </property>
  </action>
  <action name="actionStop_Shooting">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="media-playback-stop">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Stop Shooting</string>
   </property>
  </action>
  <action name="actionSequences_Editor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sequences &amp;Editor</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About Galaxy Hunter</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>A&amp;bout Qt</string>
   </property>
  </action>
  <action name="actionPause_Shooting">
   <property name="icon">
    <iconset theme="media-playback-pause">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Pause Shooting</string>
   </property>
  </action>
  <action name="actionStart_Sequences">
   <property name="icon">
    <iconset theme="media-playback-start">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Start Sequences</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>H&amp;istogram</string>
   </property>
  </action>
  <action name="actionDebayer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debayer</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
